python/CMakeLists.txt

```
@@ -23,10 +23,13 @@ project(
   # TODO: Building Python extension modules via the python_extension_module requires the C language
   # to be enabled here. The test project that is built in scikit-build to verify various linking
   # options for the python library is hardcoded to build with C, so until that is fixed we need to
-  # keep this.
+  # keep this. TODO: The C++ RMM CMake configuration targets cuda_std_17 features, which prior to
```

Suggest putting TODOs on their own lines for visibility.

```suggestion
  # keep this. 
  # TODO: The C++ RMM CMake configuration targets cuda_std_17 features, which prior to
```

---

cmake-format made this request quite difficult to accommodate, but it's a good call. I think the result makes it much clearer what each TODO is referring to.